Common/Helpers/CloudTableProvider.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.WindowsAzure.Storage.Table;
+
+namespace Microsoft.Azure.Devices.Applications.RemoteMonitoring.Common.Helpers
+{
+    public class CloudTableProvider : ICloudTableProvider
+    {
```

Comment:
do we need Provider classes? this logic could be in managers? and it could be only done once.